src/Nest/QueryDsl/Geo/GeoCoordinateJsonConverter.cs

```diff
@@ -8,7 +8,7 @@ internal class GeoCoordinateJsonConverter : JsonConverter
 		public override bool CanConvert(Type objectType) => true;
 		public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
 		{
-			var p = value as GeoCoordinate;
+			var p = (GeoCoordinate)value;
```

Comment:
Will this blow up if `value` isn't a `GeoCoordinate`?  Not sure if the null check makes sense anymore with the direct cast.